packs/feats/eyes-unclouded.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>Your senses are enhanced by your divine spark, enabling you see things as they truly are, piercing through illusions, transformations, and lies. You're affected by a constant truesight spell. Additionally, you gain a +2 circumstance bonus to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.Sense Motive] and to Perception DCs against attempts to Lie to you. When a creature attempts to Lie to you, they roll the Deception check to Lie twice and use the worse result; this is a misfortune effect.</p>"
+            "value": "<p>Your senses are enhanced by your divine spark, enabling you to see things as they truly are, piercing through illusions, transformations, and lies. You're affected by a constant @UUID[Compendium.pf2e.spells-srd.Item.Truesight] spell. Additionally, you gain a +2 circumstance bonus to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.Sense Motive] and to Perception DCs against attempts to Lie to you. When a creature attempts to Lie to you, they roll the Deception check to Lie twice and use the worse result; this is a misfortune effect.</p>"
```

Comment:
```suggestion
            "value": "<p>Your senses are enhanced by your divine spark, enabling you to see things as they truly are, piercing through illusions, transformations, and lies. You're affected by a constant @UUID[Compendium.pf2e.spells-srd.Item.Truesight] spell. Additionally, you gain a +2 circumstance bonus to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.Sense Motive] and to Perception DCs against attempts to @UUID[Compendium.pf2e.actionspf2e.Item.Lie] to you. When a creature attempts to Lie to you, they roll the Deception check to Lie twice and use the worse result; this is a misfortune effect.</p>"
```

Reply:
Might be able to use an EphemeralEffect for the roll twice on attempts to Lie to you.

Reply:
Yeah, I held off on that as I couldn't get it to work, there may be a bug. I reported it on the bugs channel a few days ago.